backend/src/index.js

```diff
@@ -7,15 +8,17 @@ const server = require('http').Server(app);
 
 const config = require('./config');
 
-// setup database connexion
 require('./config/mongoose');
 
-
 app.use(bodyParser.urlencoded({
   extended: false,
 }));
+
 app.use(bodyParser.json());
 
+// Enable CORS
+app.use(cors());
```

Comment:
Can you add something like : 
`// TODO: implement corsoptions to secure the app`
thanks

Reply:
Done.